docker_compose_test/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
       elasticsearch:
         condition: service_healthy
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost/oapi/admin/resources"]
+      test: ["CMD", "curl", "-f", "http://localhost/oapi/admin/config/resources"]
```

Comment:
The new path to check resources means that the user will have to update their docker-compose along with the new wis2box-api release. There is any way to avoid changing this path ?